`bin/makesecureboot` fails if more than one kernel is installed under `/boot`

**What happened:**
the `makesecureboot` script fails if there is more than one kernel installed in `/boot`

**Debug Logs:**
```
dracut: WARNING: running in hostonly mode in a container!!
dracut: Cannot find module directory /lib/modules/5.15.41-garden-cloud-amd64
rootfs/boot/vmlinuz-5.15.59-gardenlinux-cloud-amd64/
dracut: and --no-kernel was not specified
make: *** [Makefile:145: kvm-dev] Error 1
```

**How to reproduce it**:
build garden linux image with the `_readonly` feature enabled and install at least a second kernel.

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -143,7 +143,7 @@ The `pkg.include` file should contain the package name of the custom kernel you 
 wish to install. The `pkg.exclude` file must contain the package name of the default kernel that normally would be installed. Also, you can exclude
 any other package here you do not want in the Garden Linux image. To find the package name of the default kernel check the `pkg.include` files of the
 [cloud](/features/cloud/pkg.include), [metal](/features/metal/pkg.include) or [firecracker](/features/firecracker/pkg.include) feature, depending of
-what flavor of Garden Linux you want to build. To make a directory in the [features](/features) directory a [feature](/features/README.md), it must
+what flavor of Garden Linux you want to build. To make a directory in the [features directory](/features) a [feature](/features/README.md), it must
 contain an `info.yaml` file.
 
 <details>

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -121,9 +121,7 @@ include:
 </details> 
 
 # Customize
-The Garden Linux build pipeline offers some features to easily customize your own build of Garden Linux.
-
-
+When customizing your own build of Garden Linux you may want to add your own packages that are not in the Garden Linux repository or add your own kernel.
 
 ## Local Packages
 To install locally build packages, that are not available in the Garden Linux repository the build pipeline offers an easy way to add own packages.

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -131,7 +131,7 @@ file name of the package) to the `pkg.include` file of the feature that needs th
 
 ## Replace Kernel
 Building a Garden Linux image with more than one kernel install is not supported. In general it should work with legacy boot, but with uefi boot it
-will not be possible to choose the kernel at boot time since Garden Linux does not offer a menu for that. With the _readonly or _secureboot feature
+will not be possible to choose the kernel at boot time since Garden Linux does not offer a menu for that. With the `_readonly` or `_secureboot` feature
 enabled the image build will fail. The recommended way to use a custom kernel is to replace the default kernel.
 
 To replace the Garden Linux kernel with a custom kernel place the package with the custom kernel in the `local_packages` directory as describe in the

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -145,7 +145,7 @@ The `pkg.include` file should contain the package name of the custom kernel you 
 wish to install. The `pkg.exclude` file must contain the package name of the default kernel that normally would be installed, also you can exclude
 any other package here you do not want in the Garden Linux image. To find the package name of the default kernel check the `pkg.include` files of the
 [cloud](/features/cloud/pkg.include), [metal](/features/metal/pkg.include) or [firecracker](/features/firecracker/pkg.include) feature, depending of
-what flavor of Garden Linux your want to build. To make a directory in the [features](/features) directory a feature it must contain an `info.yaml`
+what flavor of Garden Linux you want to build. To make a directory in the [features](/features) directory a feature it must contain an `info.yaml`
 file.
 
 <details>

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -144,8 +144,9 @@ building in the [Makefile](/Makefile).
 The `pkg.include` file should contain the package name of the custom kernel you placed in the `local_packages` directory and any other package you
 wish to install. The `pkg.exclude` file must contain the package name of the default kernel that normally would be installed, also you can exclude
 any other package here you do not want in the Garden Linux image. To find the package name of the default kernel check the `pkg.include` files of the
-[cloud](/features/cloud), [metal](/features/metal) or [firecracker](/features/firecracker) feature, depending of what flavor of Garden Linux your want
-to build. To make a directory in the [features](/features) directory a feature it must contain an `info.yaml` file.
+[cloud](/features/cloud/pkg.include), [metal](/features/metal/pkg.include) or [firecracker](/features/firecracker/pkg.include) feature, depending of
+what flavor of Garden Linux your want to build. To make a directory in the [features](/features) directory a feature it must contain an `info.yaml`
+file.
 
 <details>
     <summary>Example: info.yaml</summary>

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -144,11 +144,17 @@ building in the [Makefile](/Makefile).
 The `pkg.include` file should contain the package name of the custom kernel you placed in the `local_packages` directory and any other package you
 wish to install. The `pkg.exclude` file must contain the package name of the default kernel that normally would be installed, also you can exclude
 any other package here you do not want in the Garden Linux image. To find the package name of the default kernel check the `pkg.include` files of the
-cloud, metal or firecracker feature, depending of what flavor of Garden Linux your want to build. To make a directory in the [features](/features)
-directory a feature it must contain an `info.yaml` file, a minimal example looks like this.
+[cloud](/features/cloud), [metal](/features/metal) or [firecracker](/features/firecracker) feature, depending of what flavor of Garden Linux your want
+to build. To make a directory in the [features](/features) directory a feature it must contain an `info.yaml` file.
 
-```example info.yaml
+<details>
+    <summary>Example: info.yaml</summary>
+
+```
 description: "custom changes"
 type: flag
 ```
+
+</details>
+
 For more options take a look at the [info.yaml](/features/example/info.yaml) in the example feature.

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -127,7 +127,7 @@ The Garden Linux build pipeline offers some features to easily customize your ow
 
 ## Local Packages
 To install locally build packages, that are not available in the Garden Linux repository the build pipeline offers an easy way to add own packages.
-To make a package available create the directory `local_packages` in the [root](/) of the Garden Linux directory where the `build.sh` is located.
+To make a package available create the directory `local_packages` in the root of the Garden Linux directory where the `build.sh` is located.
 Place all you own packages in that directory and add the package name (the package name is the name you would use to install it via `apt` and not the
 file name of the package) to the `pkg.include` file of the feature that needs the package.
 

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -124,7 +124,7 @@ include:
 When customizing your own build of Garden Linux you may want to add your own packages that are not in the Garden Linux repository or add your own kernel.
 
 ## Local Packages
-To install locally build packages, that are not available in the Garden Linux repository the build pipeline offers an easy way to add own packages.
+To install locally build packages, that are not available in the Garden Linux repository, the build pipeline offers an easy way to add own packages.
 To make a package available create the directory `local_packages` in the root of the Garden Linux directory where the `build.sh` is located.
 Place all you own packages in that directory and add the package name (the package name is the name you would use to install it via `apt` and not the
 file name of the package) to the `pkg.include` file of the feature that needs the package.
@@ -134,17 +134,17 @@ Building a Garden Linux image with more than one kernel install is not supported
 will not be possible to choose the kernel at boot time since Garden Linux does not offer a menu for that. With the `_readonly` or `_secureboot` feature
 enabled the image build will fail. The recommended way to use a custom kernel is to replace the default kernel.
 
-To replace the Garden Linux kernel with a custom kernel place the package with the custom kernel in the `local_packages` directory as describe in the
+To replace the Garden Linux kernel with a custom kernel, place the package with the custom kernel in the `local_packages` directory as describe in the
 [Local Packages](#local-packages) chapter. For the next steps we recommend to create your own new feature in the [features](/features) directory and
 place a `pkg.include`, `pgk.exclude` and an `info.yaml` in your feature directory. Last but not least add your new feature to the build target you are 
 building in the [Makefile](/Makefile).
 
 The `pkg.include` file should contain the package name of the custom kernel you placed in the `local_packages` directory and any other package you
-wish to install. The `pkg.exclude` file must contain the package name of the default kernel that normally would be installed, also you can exclude
+wish to install. The `pkg.exclude` file must contain the package name of the default kernel that normally would be installed. Also, you can exclude
 any other package here you do not want in the Garden Linux image. To find the package name of the default kernel check the `pkg.include` files of the
 [cloud](/features/cloud/pkg.include), [metal](/features/metal/pkg.include) or [firecracker](/features/firecracker/pkg.include) feature, depending of
-what flavor of Garden Linux you want to build. To make a directory in the [features](/features) directory a feature it must contain an `info.yaml`
-file.
+what flavor of Garden Linux you want to build. To make a directory in the [features](/features) directory a [feature](/features/README.md), it must
+contain an `info.yaml` file.
 
 <details>
     <summary>Example: info.yaml</summary>

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -9,6 +9,9 @@
 - [Package Build](#package-build)
 	- [Git Source](#git-source)
 	- [Snapshot Source](#snapshot-source)
+- [Customize] (#customize)
+  - [Local Packages] (#local-packages)
+  - [Replace Kernel] (#replace-kernel)
 
 # Introduction
 
@@ -117,4 +120,35 @@ include:
 
 </details> 
 
+# Customize
+The Garden Linux build pipeline offers some features to easily customize your own build of Garden Linux.
 
+
+
+## Local Packages
+To install locally build packages, that are not available in the Garden Linux repository the build pipeline offers an easy way to add own packages.
+To make a package available create the directory `local_packages` in the [root](/) of the Garden Linux directory where the `build.sh` is located.
+Place all you own packages in that directory and add the package name (the package name is the name you would use to install it via `apt` and not the
+file name of the package) to the `pkg.include` file of the feature that needs the package.
+
+## Replace Kernel
+Building a Garden Linux image with more than one kernel install is not supported. In general it should work with legacy boot, but with uefi boot it
+will not be possible to choose the kernel at boot time since Garden Linux does not offer a menu for that. With the _readonly or _secureboot feature
+enabled the image build will fail. The recommended way to use a custom kernel is to replace the default kernel.
+
+To replace the Garden Linux kernel with a custom kernel place the package with the custom kernel in the `local_packages` directory as describe in the
+[Local Packages](#local-packages) chapter. For the next steps we recommend to create your own new feature in the [features](/features) directory and
+place a `pkg.include`, `pgk.exclude` and an `info.yaml` in your feature directory. Last but not least add your new feature to the build target you are 
+building in the [Makefile](/Makefile).
+
+The `pkg.include` file should contain the package name of the custom kernel you placed in the `local_packages` directory and any other package you
+wish to install. The `pkg.exclude` file must contain the package name of the default kernel that normally would be installed, also you can exclude
+any other package here you do not want in the Garden Linux image. To find the package name of the default kernel check the `pkg.include` files of the
+cloud, metal or firecracker feature, depending of what flavor of Garden Linux your want to build. To make a directory in the [features](/features)
+directory a feature it must contain an `info.yaml` file, a minimal example looks like this.
+
+```example info.yaml
+description: "custom changes"
+type: flag
+```
+For more options take a look at the [info.yaml][/features/example/info.yaml] in the example feature.

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -9,9 +9,9 @@
 - [Package Build](#package-build)
 	- [Git Source](#git-source)
 	- [Snapshot Source](#snapshot-source)
-- [Customize] (#customize)
-  - [Local Packages] (#local-packages)
-  - [Replace Kernel] (#replace-kernel)
+- [Customize](#customize)
+  - [Local Packages](#local-packages)
+  - [Replace Kernel](#replace-kernel)
 
 # Introduction
 

--- a/docs/build/README.md
+++ b/docs/build/README.md
@@ -151,4 +151,4 @@ directory a feature it must contain an `info.yaml` file, a minimal example looks
 description: "custom changes"
 type: flag
 ```
-For more options take a look at the [info.yaml][/features/example/info.yaml] in the example feature.
+For more options take a look at the [info.yaml](/features/example/info.yaml) in the example feature.

